src/image-question/components/runtime.tsx

```diff
@@ -105,7 +108,7 @@ export const Runtime: React.FC<IProps> = ({ authoredState, interactiveState, set
     const inlineImage = interactiveState?.answerImageUrl || interactiveState?.userBackgroundImageUrl || authoredBackgroundUrl;
     // Render answer prompt and answer text in inline mode to replicate LARA's Image Question UI
     return <div>
-      { authoredState.prompt && <div>{renderHTML(authoredState.prompt)}</div> }
+      { authoredState.prompt && <DecorateChildren decorateOptions={decorateOptions}><div>{renderHTML(authoredState.prompt)}</div></DecorateChildren> }
```

Comment:
I would break this out into multiple lines given how much is going on in this one line at this point.

Reply:
Good call.   Looking at that again, it is pretty ugly on one line.  Fixed.